elements/pfe-dropdown/pfe-dropdown.ts

```diff
@@ -69,14 +70,16 @@ export class PfeDropdown extends LitElement {
   @observed
   @property({ type: Boolean, reflect: true }) disabled = false;
 
-  @state() private isOpen = false;
+  @observed
+  @property({ type: Boolean, reflect: true }) expanded = false;
```

Comment:
small nit: would `open` be a more semantic name? 

contrary to my point, and in your favour: https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

Reply:
I could go either way. If we use `open` then it will create a small deprecation in that `open()` is currently public method.  We also use `expanded` for accordion which is also a collapse type component."

So I suppose I would vote for `expanded`.  What say you?

Reply:
I agree with you, `expanded` is more conventional